src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+			}
+		});
+		copyButton.setToolTipText("Copy text into clipboard");
+		panel.add(copyButton);
+
+		JPanel bottomPane = new JPanel();
+		FlowLayout flowLayout = (FlowLayout) bottomPane.getLayout();
+		flowLayout.setHgap(125);
+		add(bottomPane, BorderLayout.SOUTH);
+
+		JButton submitRunButton = new JButton("Submit");
+		submitRunButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    submitSelectedJudgesSolutions();
+			}
+		});
+
+		JButton resetButton = new JButton("Reset");
+		resetButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				resetFields(true);
+			}
+		});
+		resetButton.setToolTipText("Reset back to default settings");
+		bottomPane.add(resetButton);
+		submitRunButton.setToolTipText("Submit selected sample runs");
+		bottomPane.add(submitRunButton);
+
+//		centerPane2.add(controlsPane, BorderLayout.CENTER);
+		JScrollPane leftSide = new JScrollPane(controlsPane);
+		
+		JPanel submitNoSamplesPane = new JPanel();
+		submitNoSamplesPane.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		FlowLayout fl_submitNoSamplesPane = (FlowLayout) submitNoSamplesPane.getLayout();
+		fl_submitNoSamplesPane.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane.setBounds(33, 131, 650, 32);
+		controlsPane.add(submitNoSamplesPane);
+
+		checkBoxSubmitFailingSamples = new JCheckBox("Submit Failing Samples");
+		checkBoxSubmitFailingSamples.setVerticalAlignment(SwingConstants.TOP);
+		submitNoSamplesPane.add(checkBoxSubmitFailingSamples);
+		checkBoxSubmitFailingSamples.setSelected(true);
+		checkBoxSubmitFailingSamples.setToolTipText("Submt all non-AC (Yes) submissions");
+
+		judgingTypesButtonGroup.add(submitAllJudgingTypesRadioButton);
+		submitAllJudgingTypesRadioButton.setSelected(true);
+		judgingTypesButtonGroup.add(submitSelectedJudgingTypeRadioButton);
+		
+		submitNoSamplesPane.add(submitAllJudgingTypesRadioButton);
+		submitNoSamplesPane.add(submitSelectedJudgingTypeRadioButton);
+		
+		leftSide.setPreferredSize(new Dimension(230, 230));
+		leftSide.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		leftSide.setWheelScrollingEnabled(true);
+		splitPane.setLeftComponent(leftSide);
+
+		// centerPane2.add(LogPanel, BorderLayout.SOUTH);
+		JScrollPane logScrollPanel = new JScrollPane(LogPanel);
+		logScrollPanel.setPreferredSize(new Dimension(60, 60));
+		logScrollPanel.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		logScrollPanel.setWheelScrollingEnabled(true);
+		splitPane.setRightComponent(logScrollPanel);
+		
+		
+		selectedNosLabel.setToolTipText("None selected");
+		selectedNosLabel.addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseClicked(MouseEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		submitNoSamplesPane.add(selectedNosLabel);
+		JButton selectJudementTypesButton = new JButton("Select");
```

Comment:
The GUI I see contains the following:

![image](https://github.com/pc2ccs/pc2v9/assets/9256691/8f7f9385-3c9a-4189-bf18-968b2d2fc384)

When I press the `Select` button, it shows me a dialog with the following:

![image](https://github.com/pc2ccs/pc2v9/assets/9256691/87c82ea3-a2d0-4436-80d7-dfe988269e2a)

It doesn't make sense that the panel for selecting _failed_ submissions shows me a dialog that includes _accepted_ solutions as one of the possible selection... especially when the dialog title says `Select No/Failed Judges Solutions`.

(But I do like the fact that the dialog title uses "Judges Solutions" rather than "Samples"...)